spec/views/users/index.html.erb_spec.rb

```diff
@@ -4,17 +4,29 @@
   before(:each) do
     @users = assign(:users, FactoryBot.create_list(:user, 3))
     sign_in @users.first
-
-    render
   end
 
   it "renders a list of users" do
+    render
     @users.each do |user|
       expect(rendered).to match user.email
     end
   end
 
-  it "has an + button to add a contact" do
+  it "has an + button to send contact request" do
+    render
     expect(rendered).to have_button('+', count: @users.length)
   end
+
+  it "does not have a + button for already requested user" do
+    @users.second.contact_requests << @users.first
+    render
+    expect(rendered).not_to have_css('form.button_to[action="/users/2/contact_requests"]')
```

Comment:
I think there should also be a test for the 'positive' case so that it is clear that the css selector does the right thing. Otherwise one could use any css selector here and all the tests would be green.

Reply:
I added a test - Is that what you intended?